github_pr.py

```diff
@@ -336,6 +381,8 @@ def main():
     parser.add_argument('--tableformat', default='simple', help='format of table to use')
     parser.add_argument('--noheaders', action='store_true', help='remove headers from table view. best for programmatic use of this script')
     parser.add_argument('--noratelimit', action='store_true', help="don't show the rate limit")
+    parser.add_argument('--mergeconditions', default=None, help='optional conditions to take into considersation when merging')
```

Comment:
prob is that those "choices" on line 366 are individual choices without values attached. I need a way to add the mergers file as an option.